packages/suite/src/types/coinmarket/coinmarket.ts

```diff
@@ -197,7 +203,10 @@ export interface CoinmarketBuildAccountOptionsProps extends CoinmarketGetSortedA
     tokenDefinitions: Partial<TokenDefinitionsState>;
 }
 
-export interface CoinmarketAccountOptionsGroupOptionProps extends CoinmarketCryptoListProps {
+export interface CoinmarketAccountOptionsGroupOptionProps {
+    value: CryptoId;
+    label: string; // token shortcut
+    cryptoName: string | undefined; // full name
```

Comment:
Please try to do it with self-explanatory props 🙏

Reply:
Same as comment above 
https://github.com/trezor/trezor-suite/pull/14551#discussion_r1778584950